### ISSUE TITLE
unexpected error

### ISSUE THREAD

**Issue by MichaelKubovic** [avocode/avocode#723]
Fresh install of the app, created team, created project, uploaded 2 PSD files in parallel and I've let the app upload them... the error came suddenly from the background while I wasn't using the directly.

-----
App v.2.5.3
Source UID: 50038
```
Error: invalid block type
  at Zlib._handle.onerror (zlib.js:345:17)

```

**Comment by vojtatranta** [comment]
Thank you for this report.
This error should have been fixed in 2.5.3 but it hasn't.
We are working on fix
This issue will be closed as duplicated. #507

**Comment by petrbrzek** [comment]
Hi, we've released 2.5.4 that should fix this issue. If you still have this problem in the new app, feel free to report it as a new issue
